rollup.config.js

```diff
@@ -19,6 +23,17 @@ const plugins = (es5) => [
     resolve({ browser: true }),
     typescript({ sourceMap: true, outDir: './dist' }),
     commonjs(),
+    postcss({
+        plugins: [
+            postcssImport(),
+            postcssNesting(),
+            cssnano({
+                preset: ['default', { discardComments: { removeAll: true } }],
+            }),
+        ],
+        minimize: true,
+        inject: false,
```

Comment:
am assuming that some code elsewhere is going to be handling loading the CSS file and we won't ever depend on this from e.g. array.js

Reply:
yes that's correct. I have a messy [draft PR](https://github.com/PostHog/posthog-js/pull/1923/files) that is already doing it.

it's loading the `.css` files on both `extensions/surveys.tsx` and `extensions/surveys-widget.tsx`

[here's an example](https://github.com/PostHog/posthog-js/pull/1923/files#diff-ca75b1ebdb1e4a75293f5a5f4d0bdc7b94c5ca7243b5ac9ebb49147a6dd34d83R54-R55)

Reply:
which I _think_ would also mean that we'd have to update docs for "using posthog in an extension" to say "you can't include surveys"

since Chrome store rejects extensions that depend on loading files... 🤔 but surely there's a way to use CSS in an extension...

ok, there's stuff to consider here but this is also relatively easy to change so 👍

Reply:
I think, when it's bundled, the CSS is actually loaded as a string. Then I just apply this string into `<style>` tag. so it's not really loading an external file. [I know I even had to do this check when doing tests](https://github.com/PostHog/posthog-js/pull/1923/files#diff-ca75b1ebdb1e4a75293f5a5f4d0bdc7b94c5ca7243b5ac9ebb49147a6dd34d83R604-R606) as I had to mock the CSS files out.

so I don't think we'd have this restriction

Reply:
> I think, when it's bundled, the CSS is actually loaded as a string

but (and i'm happy to learn i'm being stupid 😅 ) where is it loaded from

if I just have `array.js` and then `array.js` loads the `surveys.js` then I don't have a `surveys.css` yet, right?

so something has to load it?

Reply:
Ah, cool beans.

So, we need to make sure we use it so that it's only in the lazy loaded bundle but otherwise all cool

thank you!